cmd/bundle/init_test.go

```diff
@@ -0,0 +1,20 @@
+package bundle
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBundleInitIsRepoUrl(t *testing.T) {
```

Comment:
Could you add some test cases for incorrect input as well?

Reply:
Done!